includes/admin/class-wc-admin-post-types.php

```diff
@@ -101,6 +101,10 @@ public function setup_screen() {
 				new WC_Admin_List_Table_Products();
 				break;
 		}
+
+		// Ensure the table handler is only loaded once. Prevents multiple loads if a plugin calls check_ajax_referer many times.
+		remove_action( 'current_screen', array( $this, 'setup_screen' ) );
```

Comment:
Do we need to remove this action as well? It seems to me that we only need to remove `check_ajax_referer`. Not that removing `current_screen` will cause any issues.

Reply:
Prevents any possibility of double loading. check_ajax_referer is there so it works when you do an inline-edit (this loads in the row again via ajax).